modules/internal/ChapelDomain.chpl

```diff
@@ -793,7 +793,7 @@ module ChapelDomain {
     if isTuple(b) then return allUintTup(0);
     return false; // an error will be issued by _makeIndexTuple()
     proc allUintTup(param dim) param do
-      if dim == b.size then return false;
+      if dim == b.size then return true;
```

Comment:
Is this needed for:

> While there:, fix the bug where d by (step1:uint, step2:uint) was not producing a positive-strided domain if d was such a domain.

I don't have any reason to think this change is wrong. Just that this it looks different than your other changes dealing with test updates and deprecation warnings so it stuck out to me.

Reply:
Correct, this is the bug fix. I came across this bug as I was updating one of the tests which would not accept a domain(strides=positive) where I expected it to.